pkg/model/log.go

```diff
@@ -143,13 +149,17 @@ type LogEvent interface {
 }
 
 func ensureMaxLength(lines []logLine) []logLine {
-	bytesLeft := maxLogLengthInBytes
+	bytesSpent := 0
```

Comment:
I wonder if we should have a line limit instead of a bytes limit

Reply:
Yeah, this feels like one of those things where, given the choice, that state would be preferable, but it's not clearly worth the time to switch to that state.